Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -2591,16 +2592,14 @@ PhysicalParticleContainer::PushP (int lev, Real dt,
             const amrex::ParticleReal Bz_external_particle = m_B_external_particle[2];
 
             const std::array<amrex::Real,3>& xyzmin = WarpX::LowerCorner(box, lev, 0._rt);
+            const amrex::XDim3 gridmins = amrex::XDim3{xyzmin[0], xyzmin[1], xyzmin[2]};
```

Comment:
Same comment here as above, instead of doing
```
            const std::array<amrex::Real,3>& xyzmin = WarpX::LowerCorner(box, lev, 0._rt);
            const amrex::XDim3 gridmins = amrex::XDim3{xyzmin[0], xyzmin[1], xyzmin[2]};
```
in many places, can it be replaced with
```
            const amrex::XDim3 gridmins = WarpX::GridMins(box, lev, 0._rt);
```
or something similar?

Reply:
Yes. I've considered changing `LowerCorner` to return an `XDim3`.